GNUmakefile

```diff
@@ -64,7 +64,12 @@ terraform build provider:
 mmv1:
 	cd mmv1;\
 		bundle; \
-		bundle exec compiler.rb -e terraform -o $(OUTPUT_PATH) -v $(VERSION) $(mmv1_compile);
+		if [ "$(VERSION)" = "ga" ]; then \
+			bundle exec compiler.rb -a -e terraform -o $(OUTPUT_PATH) -v ga --no-docs; \
```

Comment:
these should use `$(mmv1_compile)` so that only specified products get generated.